### ISSUE TITLE
Read the attachment. 

### ISSUE THREAD

**Issue by PavanTejaReddy** [vijayanandrp/Outlook-Email-Wrapper#1]
Hi,
Is there a way we can read or download the email attachment. I want to parse an attachment (.csv or .xlsx) directly from mail every morning. 

Thank You

**Comment by vijayanandrp** [comment]
Yes, that is very much possible to do it. I haven't incorporated that feature in this.
